pkg/storage/chunk/client/alibaba/oss_object_client_test.go

```diff
@@ -0,0 +1,40 @@
+package alibaba
```

Comment:
Let's remove this file -- it's not regularly run and checks for keys `Prom/prometheus` which isn't guaranteed to exist. I think it's easier to remove.

Reply:
ok

Reply:
done.